tests/test_components/test_storage.py

```diff
@@ -340,3 +344,28 @@ def test_invest_content_maximum():
     assert storage_content == pytest.approx(
         [min(i * 1.9, 10) for i in range(0, 11)]
     )
+
+
+# --- BEGIN: The following code can be removed for versions >= v0.6 ---
+def test_capacity_keyword_wrapper_warning():
+    with pytest.warns(FutureWarning, match="nominal_storage_capacity"):
+        bus = solph.Bus()
+        _ = solph.components.GenericStorage(
+            nominal_storage_capacity=5,
+            inputs={bus: solph.Flow()},
+            outputs={bus: solph.Flow()},
+        )
+
+
+def test_capacity_keyword_wrapper_error():
+    with pytest.raises(AttributeError, match="nominal_storage_capacity"):
+        bus = solph.Bus()
+        _ = solph.components.GenericStorage(
+            nominal_storage_capacity=5,
+            nominal_capacity=5,
+            inputs={bus: solph.Flow()},
+            outputs={bus: solph.Flow()},
+        )
+
+
```

Comment:
I think the blank lines might be removed.

Reply:
```suggestion

```

Reply:
Black really wants those.